src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentDay = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
+
+  if (days < 3) {
+    return rentDay * days;
+  } else if (days >= 3 && days < 7) {
+    return rentDay * days - discountThreeDays;
+  }
```

Comment:
In some cases it`s better to avoid else if statement to make your code more understandable

to make your code understandable - https://medium.com/@timothydan/javascript-guard-clauses-64b999e3240